### PR TITLE
Last Check for today

### PR DESCRIPTION
**Pull request by devsTudu**

### CODE REVIEW

File: webapp.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+def gettheAddress():
+    """Read cache file and create target folder"""
+    temp = request.cookies.get('directory')
+    if temp is None:
+        address = TARGET_FOLDER
+    else:
+        address = temp
+    return address
```
Comment (prashantsengar) [inline webapp.py:19]: I don't understand this piece of code. Can you please explain this to me? I don't think setting it to TARGET_FOLDER will work? Right?

File: webapp.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    return render_template('completed.html', res=report)
+
+
+@app.route('/dashboard/deepscan')
+def deepScan():
+    """Initiate the strong arrange and redirect to report page"""
+    TARGET_FOLDER = gettheAddress()
+    destination = startwork(TARGET_FOLDER)
+    report = lib.arrange.strong_arrange(TARGET_FOLDER, destination, FOLDER_TYPES)
+    return render_template('completed.html', res=report)
```
Comment (prashantsengar) [inline webapp.py]: Can you please change the name of standardscan and deepcan everywhere to Weak Arrange and Strong arrange? We have used those names in our CLI program so it should be uniform everywhere

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+<body class="htmlNoPages">
+  <div class="content">
+  <span class="gwd-span-1yhv">Welcome To File Cleaner&nbsp;</span>
+  <p class="gwd-span-1csu">Current Directory : 
+    <button id="singlebutton" name="singlebutton" class="myButton" onclick="window.location.href='http://127.0.0.1:45201/changelocation';">Change</button>
+    <span class="gwd-span-1139">{{data['address']}}<br>
+      It has {{data['file_count']}}&nbsp; files and&nbsp;{{data['folder_count']}} folders,<br>Total items = {{data['total_items']}}<br><br><br></span></p>
+  <button id="button_1" class="gwd-button-hnfe" onclick="window.location.href='standardscan';">SimpleScan</button>
+  <button id="button_2" class="gwd-button-um3e" data-gwd-name="deepscan" onclick="window.location.href='deepscan';">Deep Scan</button><br>
+  <span class="gwd-span-s0wo">Visit our Github Page : <a href="https://github.com/prashantsengar/CleanPy" target="_blank">github.com/cleanpy</a></span></body>
```
Comment (prashantsengar) [inline templates/index.html:101]: This still shows github.com/cleanpy in the browser. Please change the text as well

File: templates/index.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    .gwd-span-s0wo {
+      width: 441px;
+      height: 43px;
+    }
+  </style>
+</head>
+
+<body class="htmlNoPages">
+  <div class="content">
+  <span class="gwd-span-1yhv">Welcome To File Cleaner&nbsp;</span>
```
Comment (prashantsengar) [inline templates/index.html:94]: > Welcome To File Cleaner&nbsp;

Please change this to show the text "CleanPy Dashboard"

Can you please also include the logo of CleanPy?

**Review by prashantsengar — COMMENTED** [review]
You did a great job @devsTudu ! 🎉

Please review these small changes and we are good to go!

**Comment by prashantsengar** [comment]
Please also see if you can also integrate Choose Directory Prompt while entering the directory instead of asking the user to write the path

File: webapp.py (hunk 1 above)
Comment (devsTudu) [reply webapp.py:19]: This is checking the cookies, and if the user has not set any different address, the installation directory is used as the target address. I have also changed the variable name at function definition from TARGET_FOLDER to TARGET & TARGET_DIR, AS it raised some warnings.

File: webapp.py (hunk 2 above)
Comment (devsTudu) [reply webapp.py]: okay no problems

File: templates/index.html (hunk 4 above)
Comment (devsTudu) [reply templates/index.html:94]: okay

File: templates/index.html (hunk 3 above)
Comment (devsTudu) [reply templates/index.html:101]: Should I use only `CleanPy`

Comment (prashantsengar) [reply templates/index.html:101]: That would be fine

**Pull request closed without merging**